app/ui-react/packages/ui/src/Data/Virtualizations/VirtualizationPublishStatus.css

```diff
@@ -0,0 +1,3 @@
+.status-label {
```

Comment:
`. virtualization-publish-status-label` maybe? The current class can easily clash.

Reply:
sounds good - I went ahead and changed it.